src/Middleware/PermissionMiddleware.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Spatie\Permission\Middleware;
+
+use Auth;
+use Closure;
+
+class PermissionMiddleware
+{
+    public function handle($request, Closure $next, $permission)
+    {
+        if (Auth::guest()) {
+            return $next($request);
+        }
+
+        if ($permission && ! Auth::user()->can($permission)) {
```

Comment:
It seems odd that the `RoleMiddleware` can accept multiple roles, but this middleware cannot.

Could you add support for multiple permissions?